opal/mca/pmix/pmix2x/Makefile.am

```diff
@@ -10,6 +10,8 @@
 # $HEADER$
 #
 
+if !OPAL_PMIX_USE_EXTERNAL
```

Comment:
autogen.subdirs is holding "libevent". So I was thinking that it's safer to keep it there.